index.php

```diff
@@ -11,10 +11,10 @@
 */
 
 $modern_fonts_config = array (
-    'id'          => 'modern-fonts-stacks',
+    'slug'          => 'modern-fonts-stacks',
     'name'        => 'Modern Fonts Stacks',
     'description' => 'Stacks of modern systems fonts, not font face assets needed. The look will vary on each system.',
-    'src'         => path_join( __DIR__, 'collection.json' ),
+    'src'         => path_join( __DIR__, 'collection-gutenberg-17.6.json' ),
```

Comment:
Noting that this change causes an error for me, since that renamed json file isn't part of the PR.

Reply:
I forgot to update the filename. That's fixed now. I updated the zip file too.